ts/parsers/graphicFrameParser.ts

```diff
@@ -50,12 +69,16 @@ export default class GraphicFrameParser {
                     }
                 }
 
-                //TODO: check if the text can have multiple values in an array, by default have seen only one so far
-                // hence getting 0th index
-                const textContent = getAttributeByPath([col], ["a:txBody", "a:p", "a:r", "a:t"])[0] || "";
+                const paragraphInfo = CheckValidObject(col, '["a:txBody"][0]["a:p"]');
+                let parsedParagraph = ParagraphParser.extractParagraphElements(paragraphInfo);
+                //edge case to handle the empty cell, without this check it will be sent as { paragraph: { content: [], ....}}
+                //and that is considered as line break in our renderer
+                if (parsedParagraph.length === 1 && isEmpty(parsedParagraph[0].content)) {
+                    parsedParagraph = [];
+                }
```

Comment:
is this exception true throughout our renderer? I am wondering if it is globally true then perhaps it can be moved to `ParagraphParser.extractParagraphElements`

Reply:
No, this is the exception for the tables only, to handle the empty cell with no text in it. 

For the textbox, if the text box is empty powerpoint by default removes that text box.

and our renderer, considers empty content as a new line, therefore we want to avoid that extra line break being added into the table columns.

Reply:
cool